shodan/__main__.py

```diff
@@ -617,22 +617,23 @@ def stats(limit, facets, filename, query):
 
 
 @main.command()
-@click.option('--color/--no-color', default=True)
```

Comment:
Why did you rearrange all these lines? With all these changes it's much harder to compare/review the PR.

Reply:
Agreed. Please put these sorts of changes in a separate PR or add a comment in the PR what changed in the available options.